index.html

```diff
@@ -145,29 +148,151 @@ <h2 class="content-home-heading" id="content-home-heading">The best destinations
 
 </section>
 <!--Sign up-->
-<section>
-<aside class="profile-bg">
-  <div class="profile" >
-  <button class="profile-user"><img src="img/user.png" alt=""></button>
-  <div class="profile-p-b">
-    <h3>Sign up and get 20% off your first booking</h3>
-    <button class="profile-sing-button">Sign up</button>
-  </div>
-  <button class="profile-clouse"><img src="img/Vector.svg" alt=""></button>
-  </div>
-</aside>
+<section class="sign-up-order">
+  <aside class="profile-bg">
+    <div class="profile" >
+      <button class="profile-user"><img src="img/user.png" alt=""></button>
+      <div class="profile-p-b">
+        <h3>Sign up and get 20% off your first booking</h3>
+        <button class="profile-sing-button">Sign up</button>
+      </div>
+      <button class="profile-clouse"><img src="img/Vector.svg" alt=""></button>
```

Comment:
Add aria-label for button
[Read more](https://developer.mozilla.org/ru/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)
Or at least alt for image instead of

Reply:
add alt